### PR TITLE
clarify "simple majority"

### PR DESCRIPTION
**Pull request by orbeckst**
As per discussion on the business list: a "simple majority" is half of the votes cast as described in link to https://en.wikipedia.org/wiki/Majority#Majority_vote. This PR just adds the link.

Please @MDAnalysis/coredevs have a look/change.

### CODE REVIEW

**Comment by lilyminium** [comment]
Given that Wikipedia can change, maybe it could just be noted in a footnote down the bottom?

**Comment by orbeckst** [comment]
Is there a nice way to do footnotes in Markdown?

File: about.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ team ("MDAnalysis Core Developers") consists of:
 
 ### Decision Making Process
 
-1. All decisions are made by simple majority of the MDAnalysis Core
+1. All decisions are made by [simple majority][] of the MDAnalysis Core
```
Comment (lilyminium) [inline about.md:80]:
```suggestion
1. All decisions are made by simple majority[^1] of the MDAnalysis Core
```

File: about.md [hunk 2]
```diff
@@ -131,3 +131,4 @@ Foundation](https://www.nsf.gov/) (as part of award ACI-1443054).
 
 
 [NumFOCUS]: https://www.numfocus.org
+[simple majority]: https://en.wikipedia.org/wiki/Majority#Majority_vote
```
Comment (lilyminium) [inline about.md:134]:
```suggestion
[^1]: We take this to mean more than half of the votes cast within a reasonable time frame. Abstentions are not included in calculating a majority vote.
```

File: about.md (hunk 1 above)
Comment (orbeckst) [reply about.md]: Thanks, just pushed very similar changes.

File: about.md (hunk 2 above)
Comment (orbeckst) [reply about.md:150]: I kept the link to the WIkipedia article in the footnote.

**Pull request merged**